lib/pluginmanager/templates/output-plugin/README.md

```diff
@@ -0,0 +1,86 @@
+# Logstash Plugin
```

Comment:
Maybe we can have a `lib/pluginmanager/templates/common` instead of copying this around?